mobile/lib/shared/views/backup_controller_page.dart

```diff
@@ -142,7 +196,7 @@ class BackupControllerPage extends HookConsumerWidget {
             BackupInfoCard(
               title: "Remainder",
               subtitle: "Images and videos that has not been backing up",
-              info: "${_backupState.totalAssetCount - _backupState.assetOnDatabase}",
+              info: "${_backupState.totalAssetCount - _backupState.assetOnDatabase}", // todo this value is not correct
```

Comment:
Oh, also while developing I noticed that the number of remaining assets to backup is found by difference (total - inDatabase) but it may be incorrect.
For example, if someone uses two devices the number of assets saved in database will be higher than the number of photos on the single device, thus showing a negative value.

Since it was not relevant to this PR I just left a comment to be addressed later